core/domain/skill_jobs_one_off.py

```diff
@@ -118,6 +118,31 @@ def reduce(key, values):
             yield (key, values)
 
 
+class DeleteInvalidSkillCommitLogEntryModelOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """This job is used to delete the invalid SkillCommitLogEntryModels.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [skill_models.SkillCommitLogEntryModel]
+
+    @staticmethod
+    def map(item):
+        # Skills don't have a 'rights' model anymore, so all commit log entries
+        # with 'rights' should be deleted.
+        if item.id.startswith('rights') or item.post_commit_status == 'private':
```

Comment:
What happens to the corresponding SkillModels then? Will some entries have missing commit log entry models (since you're deleting stuff with `item.post_commit_status == 'private'` as well)?